apps/server/src/modules/loadStyles.ts

```diff
@@ -0,0 +1,26 @@
+import { copyFileSync, existsSync } from 'fs';
+import { pathToStartStyles, resolveStylesDirectory, resolveStylesPath } from '../setup.js';
+import { ensureDirectory } from '../utils/fileManagement.js';
+import { reportSentryException } from './sentry.js';
+
+/**
+ * @description ensures directories exist and populates database
+ * @return {string} - path to db file
+ */
+const populateStyles = () => {
+  const stylesInDisk = resolveStylesPath;
+  ensureDirectory(resolveStylesDirectory);
+
+  // if dbInDisk doesn't exist we want to use startup db
```

Comment:
Looks good, just a few copy-paste artifacts
`path to db file`
`if dbInDisk doesn't exist we want to use startup db`